handbook/product/user_research/user_research_observer.md

```diff
@@ -0,0 +1,39 @@
+# So you’re about to help us with user testing
+
+Thank you so much for helping us with our user research! Observing and notetaking during user interviews and testing sessions is an amazing way for you to get a first-hand look at how others use our products and ideas, and your role as a notetaker is a huge help for the team.
+
+Here’s everything you need to know ahead of time.
+
+## Prerequisites
+
+- We use Lookback to conduct the sessions. Lookback requires Chrome, so please make sure you have Chrome installed and ready.
+
+## What to expect
+
+- You will receive a link to join a session as an **observer** ahead of time. You can join anytime before the session begins.
+- As an observer, you will not be on camera or audio, and the participant will not have any indication that you’re observing the session. The participant will only interact with the moderator.
+- While the session is underway, there’s two things you can do: use the chat to talk with the moderator and other observers, and use the notetaking section to take notes.
+- All of your notes will be timestamped at the moment you begin typing.
+
+## How to take really good notes
```

Comment:
Do we want to make it sound like taking notes is mandatory? Or it should be an extra? @quinnkeast

Reply:
I would really, really prefer if it were semi-mandatory. Every observer has a different perspective that will give us a more complete understanding of the session. The notes don't have to be extensive, but even _some_ observations are great.

Also – we ought eventually include links to example sessions with good notes, so people can see exactly how it's done.